packages/lesswrong/components/ea-forum/voting-portal/VotingPortalThankYou.tsx

```diff
@@ -214,34 +228,37 @@ const VotingPortalThankYou = ({currentUser, classes}: {
             </div>
           </div>
         </div>
-        <div className={classes.button} role="button" onClick={toggleFlair}>
+        <div className={classNames(classes.button, {
+              [classes.outlineButton]: currentUser.givingSeason2023VotedFlair,
+            })} role="button" onClick={toggleFlair}>
           {loadingFlair
             ? <Loading white />
             : <><ForumIcon icon="ArrowRight" className={classes.arrowIcon} /> {currentUser.givingSeason2023VotedFlair
-              ? "Remove icon from your profile"
+              ? "Remove icon from profile"
               : "Add icon to your profile"}</>
           }
         </div>
       </div>
       <div className={classes.hr} />
-      <ul className={classes.list}>
-        <li>
+      <div className={classes.list}>
+        <div className={classes.listRow}>
+          -&gt;
```

Comment:
Uhhhh. We have a bleeping ligature for this? I mean, if we have it, I guess use it, but seems kinda insane.

Reply:
Yeah Ollie introduced me to this, and it is kind of insane. It only seems to work in some elements (e.g. it doesn't work in buttons), I'm on the fence about using it in general but in this case the "ArrowRight" icon looked too narrow whereas this is just right so I decided to go with it